gh-actions/common/run-autopkgtest/action.yml

```diff
@@ -0,0 +1,172 @@
+name: Run autopkgtest for debian packages
+description: |
+  Runs autopkgtest for a built source package, optionally using
+  already built deb files.
+  The build happens in an LXD container so that autopkgtest can setup all the
+  required parameters to run the test with all the expected constraints.
+
+inputs:
+  source-changes:
+    description: |
+      The path to the source .changes file.
+      The the rest of the source files are expected to be in the same directory
+    required: true
+
+  debs-dir:
+    description: |
+      Directory where the deb files are located.
+      It will be the same of the source-dir if not provided.
+    required: false
+
+  lxd-image:
+    description: |
+      The LXD image used to build the package.
+      A docker-style image aliases can also be used.
+    required: false
+    default: ubuntu:rolling
+
+  autopkgtest-args:
+    description: Additional arguments to pass to autopkgtest
+    required: false
+    default: ''
+
+runs:
+  using: "composite"
+  steps:
+    - name: Check prerequisites
+      shell: bash
+      run: |
+        echo "::group::Check prerequisites"
+        set -eux
+
+        sudo apt install dctrl-tools
```

Comment:
missing `-y`, no?

Reply:
same of https://github.com/canonical/desktop-engineering/pull/58#discussion_r1918549735